hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestCopyOnWriteRollbackActionExecutor.java

```diff
@@ -106,7 +107,7 @@ public void testCopyOnWriteRollbackActionExecutorForFileListingAsGenerateFile()
         .withBaseFilesInPartition(p1, "id21").getLeft()
         .withBaseFilesInPartition(p2, "id22").getLeft();
 
-    HoodieWriteConfig writeConfig = getConfigBuilder().withRollbackUsingMarkers(false).build();
+    HoodieWriteConfig writeConfig = getConfigBuilder().withRollbackUsingMarkers(false).withEmbeddedTimelineServerEnabled(false).build();
```

Comment:
Should the test be irrelevant to timeline server?

Reply:
and why the COW table test relevent with log file marker?

Reply:
Yeah I agree, it should not be required for log file marker. Removing the changes for this test.